test/compile/mark/bar.test.ts

```diff
@@ -43,9 +43,9 @@ describe('Mark: Bar', () => {
     expect(props.height).toBeUndefined();
   });
 
-  it('should draw vertical bar, with y from zero to field value and bar with quantitative x, x2, and y', () => {
-    const x: PositionFieldDef<string> = {field: 'bin_start', type: 'quantitative'};
```

Comment:
Since there are the test cases for the bin, I change this to cover not-binned cases.